snap/snapcraft.yaml

```diff
@@ -0,0 +1,21 @@
+name: perceval
+version: '0.8.0' 
+summary: Retrieve and gather data from software repositories, 
+description: |
+  Package that gathers and retrieves data from known software repositories.
+  Repository sources includes askbot, bugzilla, bugzillarest, confluence, 
+  discourse, dockerhub, gerrit, git, github, gname, hyperkitty, jenkins, jira, 
+  mbox, mediawiki, meetup, nntp, phabricator, pipermail, redmine, rss, slack, 
+  stackexchange, supybot, telegram.
+grade: stable 
+confinement: strict 
+
+parts:
+  perceval:
+    source: https://github.com/grimoirelab/perceval.git
```

Comment:
When the snapcraft.yaml is upstream, there is no need to specify the full path to the repo, because you are already in a directory with the source downloaded. Please replace it with:

`source: .`